### DISCUSSION TITLE
Release schedule

### DISCUSSION THREAD

**Discussion by jtl1990** [encode/databases#481] [Q&A]
Hi all,

Just wondering what the release schedule is like.
We are hoping to get access to the changes in https://github.com/encode/databases/pull/436 to swap from `asyncpg` to `aiopg`. 

Thanks in advance!

**Comment by zevisert** [comment]
Also curious about an answer to this. I came here ready to make a PR to fix `@db.transaction` typing, but it's already done in #493 -- but that hasn't been released yet
